deploy-agent/tox.ini

```diff
@@ -1,6 +1,7 @@
 [tox]
-envlist=py27,py37,py38,py39
+envlist=py27,py37,py38,py39,black
 recreate=True
+skip_missing_interpreters=True
```

Comment:
so when we run tox locally if a python version isn't installed it doesn't throw an error